packages/react-core/src/components/EmptyState/EmptyState.tsx

```diff
@@ -15,7 +15,7 @@ export interface EmptyStateProps extends React.HTMLProps<HTMLDivElement> {
   className?: string;
   /** Content rendered inside the EmptyState */
   children: React.ReactNode;
-  /** Modifies EmptyState max-width */
+  /** Modifies EmptyState max-width and sizes of EmptyStateIcon, EmptyStateTitle and EmptyStateBody */
   variant?: 'xs' | 'small' | 'large' | 'xl' | 'full';
```

Comment:
Since this is breaking change PR.  Can we go ahead and change the `small` variant ti `sm` and `large to `lg`.  That is more consistent with our naming elsewhere.